frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -289,6 +293,16 @@
           {$t('project_page.last_commit')}:
           <span class="text-secondary"><FormatDate date={project.lastCommit} /></span>
         </div>
+        {#if project.type === ProjectType.FlEx}
+        <div>
+          {$t('project_page.num_entries')}:
+          <span class="text-secondary">
+            {#await $lexEntryCount then num_entries}
```

Comment:
because this is awaited it'll never show from SSR, can we use the gql response directly if it exists?

Reply:
Yes, though the code might become more complicated. I had the store return an already-resolved Promise if the GQL response existed so that the "shape" of the result will always be the same.

What I might do is to say if it's a promise, await it, but if it's a plain value, display it. Or even make a component to encapsulate that behavior so we can reuse it: accept a `Promise<T> | T` and either display or await it accordingly.